test/e2e/app-dir/segment-cache/basic/app/cache-life-seconds/page.tsx

```diff
@@ -0,0 +1,15 @@
+import { unstable_cacheLife } from 'next/cache'
+
+export default async function CacheLifeSecondsPage() {
+  'use cache'
+  unstable_cacheLife('seconds')
```

Comment:
```suggestion
  unstable_cacheLife({ stale: 0, revalidate: 1, expire: 60 })
```
We're about to change `'seconds'` to have a 30s stale time: https://github.com/vercel/next.js/pull/82332 
this will make the test ineffective. maybe let's just inline a stale time of 0 since that's what we're guarding against here?